src/components/CursorPaginationManager/CursorPaginationManager.tsx

```
@@ -0,0 +1,328 @@
+          <div key={index}>{renderItem(item, index)}</div>
+        ))}
+      </div>
+      {pageInfo?.hasNextPage && (
+        <div className={styles.loadMoreSection}>
+          <button
+            onClick={handleLoadMore}
+            disabled={isLoadingMore}
+            className={styles.loadMoreButton}
+            aria-label={t('loadMoreItems')}
+            data-testid="load-more-button"
+          >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)